cmd/meroxa/root/apps/deploy.go

```diff
@@ -271,6 +271,9 @@ func createTarAndZipFile(src string, buf io.Writer) error {
 	tarWriter := tar.NewWriter(zipWriter)
 
 	err = filepath.Walk(appDir, func(file string, fi os.FileInfo, err error) error {
+		if err != nil {
+			return err
```

Comment:
returning this error will halt the tree walk, this is intended right?

Reply:
I figured it should be, otherwise we could get nil pointer errors when trying to operate on a file that isnt there.